src/components/content/content.ts

```diff
@@ -400,6 +404,8 @@ export class Content extends Ion implements OnDestroy, OnInit {
    */
   ngOnDestroy() {
     this._scLsn && this._scLsn();
+    this._viewCtrlReadSub && this._viewCtrlReadSub.unsubscribe();
+    this._viewCtrlWriteSub && this._viewCtrlWriteSub.unsubscribe();
     this._scroll && this._scroll.destroy();
```

Comment:
Can you add this._viewCtrlReadSub = this._viewCtrlWriteSub = null?
just for completeness